### ISSUE TITLE
Using custom labels for a MSCOCO dataset, but it still looks for the standard ones

### ISSUE THREAD

**Issue by Xonxt** [AlexeyAB/darknet#8434]
**tl;dr**
Looking at other issues, it seems like the Darknet code will always try to look for the word `/images/` in the training image paths and replace it with `/labels/`. There should be a way to disable that, without changing the training code and rebuilding the whole solution.

**More detail:**
I'm using the MS COCO dataset and have the images with the same folder structure (i.e. `my_datasets/coco/images/train2017/*.jpg`), but I've provided my custom labels in `*.txt` files, putting them into the same folder as the images.

But when running `darknet.exe detector train ...` it still tries to look for the standard MS COCO labels in `my_datasets/coco/labels/train2017/*.txt`, completely ignoring my custom labels. I had to rename the `/images/` to something else to avoid that.
